resources/party.ts

```diff
@@ -21,9 +24,17 @@ export default class PartyTracker {
     this.idToName_ = {};
 
     // role -> [names] but only for party
-    this.roleToPartyNames_ = {};
+    this.roleToPartyNames_ = {} as Record<Role, string[]>;
     for (const role of Util.getAllRoles())
       this.roleToPartyNames_[role] = [];
+  }
+
+  // Bind this to PartyChanged events.
+  onPartyChanged(e: { party: Party[]}): void {
```

Comment:
will it be better to add a types/`events.d.ts` for all events?

Reply:
Those are typed for OverlayPlugin/Cactbot backend, I think to put them in the `global` file is well, not neccesary to create many files? They are not very long, even fill up all the types later.